packages/@expo/cli/src/start/server/type-generation/__typetests__/generateFixtures.ts

```diff
@@ -1,46 +1,38 @@
+import { getTypedRoutesDeclarationFile } from 'expo-router/build/typed-routes/generate';
 import { writeFile } from 'fs/promises';
 import { join } from 'path';
 
-import { getTemplateString } from '../routes';
-
-type Fixture = {
-  staticRoutes: string[];
-  dynamicRoutes: string[];
-  dynamicRouteTemplates: string[];
-};
+function getMockContext(context: string[]) {
```

Comment:
We don't need a real context here. The helpers inside Expo Router are not publicly exposed, and trying to access them is a commonjs/esm hassle.